src/main/scala/edu/cmu/cs/obsidian/codegen/CodeGenYul.scala

```diff
@@ -116,14 +117,11 @@ object CodeGenYul extends CodeGenerator {
         // there's anything else that returns ever.
         val retExpr = FunctionCall(
             Identifier("return"),
-            Seq(Literal(LiteralKind.number,"0","int"),Literal(LiteralKind.number,"0","int"))
-        )
-        statement_seq_deploy = statement_seq_deploy :+ ExpressionStatement(retExpr)
-        statement_seq_runtime = statement_seq_runtime :+ ExpressionStatement(retExpr)
+            Seq(Literal(LiteralKind.number,"0","int"),Literal(LiteralKind.number,"0","int")))
 
         // create runtime object
         val runtime_name = contract.name + "_deployed"
-        val runtime_obj = YulObject(runtime_name, Code(Block(statement_seq_runtime)), Seq(), Seq())
```

Comment:
Scala style guide says to leave these spaces in: https://docs.scala-lang.org/style/method-invocation.html